app/routes/events/view/speakers/create.js

```diff
@@ -0,0 +1,40 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  titleToken() {
+    return this.get('l10n').t('Create session');
+  },
+  async model() {
+    const eventDetails = this.modelFor('events.view');
+    return {
+      event : eventDetails,
+      form  : await eventDetails.query('customForms', {
+        'page[size]' : 50,
+        sort         : 'id'
+      }),
+      session: await this.get('store').createRecord('session', {
+        event   : eventDetails,
+        creator : this.get('authManager.currentUser')
+      }),
+      sessions: await this.get('store').query('session', {
```

Comment:
We need to list the sessions of the event we are adding the speaker or session for. we must query `eventDetails` here to fetch all the sessions of the event.

Reply:
@geekyd but I am getting sessions of that event only. As in the next line I have specified event as eventDetails.

Reply:
This query is returning sessions that do not belong to the event.

Reply:
@geekyd oh sorry! did it finally please review.